### ISSUE TITLE
Screen goes black when casting vote.

### ISSUE THREAD

**Issue by dafuga** [greymass/eos-voter#83]
The voting appears to work though. This is the error that appears in the console.

```Failed prop type: producers.lastError must be an object

Expected: {
  
}

Actual Value: false

Actual Type: boolean

    in Producers (created by BasicVoterContainer)
    in BasicVoterContainer (created by Connect(BasicVoterContainer))
    in Connect(BasicVoterContainer) (created by Route)
    in Route (created by withRouter(Connect(BasicVoterContainer)))
    in withRouter(Connect(BasicVoterContainer)) (created by Route)
    in Route (created by _default)
    in Switch (created by _default)
    in Router (created by HashRouter)
    in HashRouter (created by _default)
    in _default (created by Root)
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Root)
    in Provider (created by Root)
    in I18nextProvider (created by Root)
    in Root
    in e

**Comment by aaroncox** [comment]
We are going to have to get a handle on the props validation and make sure those are all set right. That's probably what a lot of the black screen modals after submitting are.

**Comment by aaroncox** [comment]
This should be resolved in 0.1.6.
